scripts/helperFiles/getGhostConfig.js

```diff
@@ -6,7 +6,7 @@ let databaseConfig;
 try {
   databaseConfig = fs.readFileSync(__dirname+'/../../config/database.config.js', 'utf8');
     // removes the export default and last 2 characters '`;'
-    databaseConfig = databaseConfig.substring(15, databaseConfig.length-2);
+  databaseConfig = databaseConfig.substring(15, databaseConfig.length-1);
```

Comment:
Because I'm quite everything was working here before, if you found a bug please elaborate and we'll talk about it.